### PR TITLE
fix: initializeSb check and run workflow only in base repository

### PR DESCRIPTION
**Pull request by natterstefan**
# Story

- fixes issue with the release GitHub Action that runs in forks and causes forks to have failing checks
- fix an issue with `initializeSb` checking `roam42.wB` instead of `roam42.smartBlocks`

## Screenshots

image

### CODE REVIEW

**Comment by natterstefan** [comment]
Hey @dvargas92495, please let me know what you think.

File: main.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ if( typeof window.roam42 == 'undefined' ) {
         try { roam42.typeAhead.loadTypeAhead(); } catch(e){};
         try { roam42.quickRef.component.initialize(); } catch(e){};
         const initializeSb = (counter) => setTimeout(()=> { 
-          if (roam42.wB) {
+          if (roam42.smartBlocks) {
```
Comment (dvargasfuertes) [inline main.js:135]: embarrassing

**Review by dvargasfuertes — APPROVED** [review]
Thanks!

**Comment by dvargasfuertes** [comment]
oh wow sorry, I see that the PR was created 3 days ago, but I only got the notification today after the comment. not sure why I didn't receive a notification on PR creation

**Comment by natterstefan** [comment]
Don't worry @dvargas92495, maybe some notification settings are not working atm. Thanks for merging it. 👍

**Pull request merged**